universal-application-tool-0.0.1/app/views/applicant/ApplicantProgramBlockEditView.java

```diff
@@ -274,6 +275,27 @@ private Tag renderReviewButton(Params params) {
         .withClasses(ApplicantStyles.BUTTON_REVIEW);
   }
 
+  private Tag renderPreviousButton(Params params) {
+    int previousBlockIndex = params.blockIndex() - 1;
+    String redirectUrl;
+
+    if (previousBlockIndex >= 0) {
+      redirectUrl =
+          routes.ApplicantProgramBlocksController.previous(
+                  params.applicantId(), params.programId(), previousBlockIndex, params.inReview())
+              .url();
+    } else {
+      redirectUrl =
```

Comment:
My understanding is that this should be directing to the Preview page. Can you help me understand how this brings the applicant to the preview page rather than the final review page?

Reply:
Ah yes - it should have been "preview", thanks for calling this out! Fixed.